torchio/transforms/preprocessing/intensity/projection.py

```diff
@@ -0,0 +1,158 @@
+import torch
+from torchio.data.image import ScalarImage
+from ....data.subject import Subject
+from ...intensity_transform import IntensityTransform
+from typing import Optional
+from math import ceil
+
+
+class Projection(IntensityTransform):
+    """Project intensities along a given axis, possibly with sliding slabs.
+
+    Args:
+        axis: Possible inputs are ``'Left'``, ``'Right'``, ``'Anterior'``,
```

Comment:
For many images, anatomical axes don't make sense. It should be possible to pass the axis index instead (0, 1 or 2). I'll add a comment on this below*